tests/validations.sh

```diff
@@ -80,6 +114,19 @@ cd ..
 
 # *** TODO: Get ride of duplication
 
+# First we should check hld pipelines exist. If there is a pipeline with the same name we should delete it
+hld_pipeline_exists $AZDO_ORG_URL $AZDO_PROJECT $hld_dir $manifests_dir
+
+# Create the hld to manifest pipeline
+echo "hld_dir $hld_dir"
+echo "hld_repo_url $hld_repo_url"
+echo "manifest_repo_url $manifest_repo_url"
+spk hld install-manifest-pipeline -o $AZDO_ORG -d $AZDO_PROJECT -p $ACCESS_TOKEN_SECRET -r $hld_dir -u https://$hld_repo_url -m https://$manifest_repo_url
```

Comment:
@mtarng why the `https://` in front of `$manifest_repo_url`? Is it required?

Reply:
Yeah, unfortunately, the node library [git-url-parse](https://www.npmjs.com/package/git-url-parse) that we're using to parse doesn't recognize something like `dev.azure.com/bhnook/fabrikam/_git/hld` as a valid git url, so we prepend `https://` to the url. That's also why i've updated the default spk-config.yaml values for `azure_devops` repo values.

Reply:
@mtarng This seems like a bug we should fix. This is more overhead for users. If a bug doesn't exist I can create one